demo-data/src/main/java/org/jboss/pnc/demo/data/DatabaseDataInitializer.java

```diff
@@ -131,10 +132,15 @@ public void initiliazeData() {
                     .name("JBoss JavaEE Servlet Spec API").description("JavaEE Servlet Spec API")
                     .projectUrl("https://github.com/jboss/jboss-servlet-api_spec")
                     .issueTrackerUrl("https://issues.jboss.org/browse/JBEE").build();
+            Project project4 = Project.Builder.newBuilder()
```

Comment:
I think we may try to remove some of those projects and start replacing it with proper ones (like Fabric8)